src/main/java/org/protege/editor/owl/client/LocalHttpClient.java

```diff
@@ -61,10 +70,16 @@
 
 	private static final String authHeader = "Authorization";
 
+	private static final String SNAPSHOT_CHECKSUM = "-checksum";
+
 	private final String serverAddress;
 
 	private final OkHttpClient httpClient;
 
+	private final ListeningExecutorService service = MoreExecutors.listeningDecorator(Executors.newSingleThreadExecutor());
+
+	private ProgressDialog dlg = new ProgressDialog();
```

Comment:
Can we scope these instance variables to the methods where they're being used?

The ProgressDialog initialization fails for some unit tests for example.